### PR TITLE
Add kernel meta examples

### PR DESCRIPTION
**Pull request by avramidis**

### CODE REVIEW

**Comment by avramidis** [comment]
Still WIP until the data are merged

File: examples/meta/src/kernel/anova.sg [hunk 1]
```diff
@@ -0,0 +1,20 @@
+File traindat = csv_file("../../data/fm_train_real.dat")
+File testdat = csv_file("../../data/fm_test_real.dat")
+
+#![create_features]
+Features feats_train = features(traindat)
+Labels feats_test = features(testdat)
+#![create_features]
+
+#![create_kernel]
+Kernel anova_kernel = kernel("ANOVAKernel")
```
Comment (karlnapf) [inline examples/meta/src/kernel/anova.sg]: could you use the new kwargs style here?
`Kernel k = kernel("ANOVEKernel", cardinality=2)`

File: examples/meta/src/kernel/anova.sg [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+anova_kernel.put("cardinality", 2)
+anova_kernel.put("cache_size", 10)
+anova_kernel.init(feats_train, feats_train)
+#![create_kernel]
+
+#![get_kernel_features]
+RealMatrix km_test_0 = anova_kernel.get_kernel_matrix()
```
Comment (karlnapf) [inline examples/meta/src/kernel/anova.sg]: these names are a bit odd. What about `K_train_train` and `K_train_test`

File: examples/meta/src/kernel/chi2.sg [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+#![create_features]
+Features feats_train = features(traindat)
+Labels feats_test = features(testdat)
+#![create_features]
+
+#![create_kernel]
+Kernel chi2_kernel = kernel("Chi2Kernel")
+chi2_kernel.put("width", 1.4)
+chi2_kernel.put("cache_size", 10)
```
Comment (karlnapf) [inline examples/meta/src/kernel/chi2.sg]: no need to put those in the examples

**Comment by avramidis** [comment]
I made the changes but I cannot change the status of the PR to ready for review. I guess only members can do that?

**Comment by karlnapf** [comment]
It’s your or so I think you have to do that. Seems I can merge anyways, will check it biw

**Comment by avramidis** [comment]
Now I can see the "Ready for review" button. Probably Github was glitching yesterday.

File: examples/meta/src/kernel/circular.sg [hunk 4]
```diff
@@ -0,0 +1,19 @@
+File traindat = csv_file("../../data/fm_train_real.dat")
+File testdat = csv_file("../../data/fm_test_real.dat")
+
+#![create_features]
+Features feats_train = features(traindat)
+Features feats_test = features(testdat)
+#![create_features]
+
+#![create_kernel]
+Distance d = distance("EuclideanDistance", lhs=feats_train, rhs=feats_train)
```
Comment (karlnapf) [inline examples/meta/src/kernel/circular.sg]: Can you mark left and right hand side parameters as read only? And then we force users to use the unit function, just like done in CKernel.
Sorry I didn’t say before we just added this option

**Comment by avramidis** [comment]
Changing the left and right hand parameters as read only break a lot of distance examples. @karlnapf better to do that in another PR.

**Comment by karlnapf** [comment]
Conflicts... cannot merge

**Comment by avramidis** [comment]
Ok. I will probably do it tomorrow.

**Comment by avramidis** [comment]
Should be fine now

**Pull request merged**